TACC/Core-Portal
# Task/fp 318 job history modal  more data

# Notice: #

Keeping as Draft. See https://github.com/TACC/Frontera-Portal/pull/99#pullrequestreview-462385586.

- `task/FP-318-job-history-modal` #99
    - `task/FP-318-job-history-modal--ui-tweaks-more-date-expand-tweaks` #143
        - `task/FP-318-job-history-modal--ui-tweaks` #140
        - `task/FP-318-job-history-modal--expand-tweaks` #141
        - `task/FP-318-job-history-modal--more-data` #142
        - `task/FP-318-job-history-modal--re-style` #144

## Overview: ##

_(via `task/FP-318-job-history-modal`; see PR #99)_

Add input data that is not shown.

## PR Status: ##


## Related Jira tickets: ##

* [FP-318](https://jira.tacc.utexas.edu/browse/FP-318)

## Summary of Changes: ##

_(each change is a single commit)_


- Change "FailureReport": "Failure Report" to "Failure Report": "System Output".
    Move the rest of the data to "Config" section.\*
    Reorder "Config" values.\*
- Add "Output" data.\*

_\* So that user can review all entered data with the same organization in which they entered the data (apps section job form). This modal is for debugging, so redundancy is okay._

## Testing Steps: ##
1. Open Job History Detail View modal.
2. Ensure no data is lost.
3. See data is re-organized to match application form sections.
4. See that "Output" section is added and has accurate data.

## UI Photos:

[source of categorization: Applications section job submission form](https://user-images.githubusercontent.com/62723358/89514068-fa26be80-d79a-11ea-89a1-37d8988d87c8.png)
<img width="806" alt="FP-318 More Data 2" src="https://user-images.githubusercontent.com/62723358/89520268-39590d80-d7a3-11ea-8f28-cc1fcefd0795.png">
<img width="806" alt="FP-318 More Data 1" src="https://user-images.githubusercontent.com/62723358/89520272-3b22d100-d7a3-11ea-9c46-0791e15b79ad.png">

## Notes: ##
